applications/CoSimulationApplication/python_scripts/base_classes/co_simulation_data_transfer_operator.py

```diff
@@ -0,0 +1,29 @@
+from __future__ import print_function, absolute_import, division  # makes these scripts backward compatible with python 2.6 and 2.7
+
+# Importing Kratos
+import KratosMultiphysics as KM
+
+class CoSimulationDataTransferOperator(object):
```

Comment:
Would be nice to add a short description of the purpose of this class.

Reply:
done in #5187